Makefile

```diff
@@ -46,16 +46,16 @@ ifeq (${VERBOSE}, 1)
 	go env
 endif
 
-	go build ${GOARGS} -tags "${GOTAGS}" -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/${BINARY_NAME} .
+	go build ${GOARGS} -tags "${GOTAGS}" -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/${BINARY_NAME}_${VERSION} .
```

Comment:
I don't really see the added value here. This is mostly helping local development, so version will most probably be a branch name?

Reply:
Maybe you are right and I am feature creeping this...
I was looking for a way to use version during my test

provider "k8s" {
  version = "0.4.0"
  kubeconfig = "./kubeconfig_tools-internal"
}

also seen an open issue to publish the provider on terraform 
https://github.com/banzaicloud/terraform-provider-k8s/issues/6

and I guess that would help to solve the issue. The 4 fields form to add the provider to community provider looks pretty straightforward! 

https://www.terraform.io/docs/providers/type/community-index.html

In any case, let me know if you want me to split this in another PR, maybe there is more works to do in order to have this working on the terraform side.

Reply:
Production binaries are built using goreleaser, so this won't really have an effect. I would prefer doing work on that issue in a spearate PR